### PR TITLE
News pattern

### PR DESCRIPTION
**Pull request by trevorsaint**
Added pattern for News pages to include:

- Landing
- Article
- Category
- Tag

### CODE REVIEW

**Review by boxadesign — DISMISSED** [review]

**Comment by trevorsaint** [comment]
Awaiting content from @LauraC119

**Comment by trevorsaint** [comment]
[News [Pages]](https://trello.com/c/N5HRUKea/1128-news-pages)

**Comment by trevorsaint** [comment]
> **A reviewer left a comment:**
> 
> "A list of categories appears on the right-hand side of the landing page.." - does this need some instruction/link to related content?
> 
> Browser metadata
> [Open Deploy Preview](https://deploy-preview-1515--ons-design-system.netlify.app/patterns/news/) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=60b65a8489e4a60007158f9d&commentID=60b8af13b12c92fb975eb07c&resolution=resolved)

I have added reference to related content and also our follow us pattern.

**Review by LauraC119 — DISMISSED** [review]
Looks good now those changes made. thanks

File: src/patterns/news/index.njk [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+Tags are not mandatory. Do not use the Tags block if there are no tags. A tag helps to identify an article, for example, “Census 2021”. Good tags help improve search performance. They need to be an important word or phrase that users will recognise. 
+
+You can use more than one tag for each article. Each tag is highlighted in yellow on the landing page summary. An “All posts [tag name]” landing page lists all the news posts that use that tag on the website. 
+
+{{
+    patternlibExample({ "path": "patterns/news/examples/tag/index.njk" })
+}}
+
+### Category
```
Comment (jrbarnes9) [inline src/patterns/news/index.njk:87]: Assuming tags are only visible at the article level, I'd be inclined to bump the 'categories' section up above anything related to the article page.

File: src/patterns/news/examples/tag/index.njk [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+        ]
+    },
+    "footer": {}
+} %}
+
+{% block main %}
+
+<div class="grid">
+    <div class="grid__col col-7@m">
+        <h1 class="u-fs-xxl">All posts for Census2021</h1>
```
Comment (jrbarnes9) [inline src/patterns/news/examples/tag/index.njk:39]: I think it would help if we wrapped the tag word in a highlight span here too so the connection between the tag elements is clear

File: src/patterns/news/examples/tag/index.njk [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+                        "title": 'Census 2021 myth buster – 11 things you think you know about the census but you’re wrong',
+                        "date": {
+                            "short": '7 April 2021',
+                            "iso": '2021-04-07'
+                        },
+                        "category": {
+                            "title": 'Press releases',
+                            "url": '#0'
+                        },
+                        "excerpt": 'The Office for National Statistics dispels some common myths around <mark class="highlight">Census 2021</mark>Census 2021</mark>.'
```
Comment (jrbarnes9) [inline src/patterns/news/examples/tag/index.njk:75]:
```suggestion
                        "excerpt": 'The Office for National Statistics dispels some common myths around <em class="highlight">Census 2021</em>.'
```
Should use `em` to be consistent with the “See important information” pattern

File: src/patterns/news/index.njk (hunk 1 above)
Comment (jrbarnes9) [reply src/patterns/news/index.njk:87]: I would make this the first sub-heading in 'How to use this pattern' after describing how to use the landing page, and remove the example.  

So you'd have:
- Pattern title
- Lead in line
- [Main news page example]
- When to use this pattern
- How to use this pattern
- Categories
- [Category list page example]
- News articles
- [News article page example]
- Tags
- [Tag list page example]

File: src/patterns/news/index.njk [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+
+How to display, tag, categorise and provide a landing page preview for news articles.
+
+{{
+    patternlibExample({ "path": "patterns/news/examples/landing/index.njk" })
+}}
+
+## When to use this pattern
+
+Use this pattern to upload a news article to a public-facing website.
```
[the remark this answers is not in the document]
Comment (LauraC119) [reply src/patterns/news/index.njk:18]: yep, I agree. Can you change to publish please @trevorsaint

File: src/patterns/news/index.njk [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+- summary
+- body text
+- category
+- date
+
+Use the [articles component](/components/articles) to build consistent articles.
+
+The content needs to be clearly written and as short as possible. Use plain language and “frontload” – put the most important word or points first. This ensures that if the user is skimming the article, they read the most important thing and understand it. 
+
+Our house style guide has [advice on writing in plain English](https://style.ons.gov.uk/category/writing-for-the-web/plain-english). You could also ask a content designer to help you rewrite the content so it is clear and helpful for the user.
```
[the remark this answers is not in the document]
Comment (LauraC119) [reply src/patterns/news/index.njk:34]: Yes, was thinking that this was perhaps too much detail here. It will work with just a mention of the articles component here (as we do on other patterns) and I'll keep all the detail for there

File: src/patterns/news/index.njk [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+
+### Landing page
+
+This page sorts the already-published news so that the most recently posted article is at the top of the list.
+
+Each landing page preview needs to have a:
+
+- short, clearly written title that links to the news article
```
[the remark this answers is not in the document]
Comment (LauraC119) [reply src/patterns/news/index.njk:42]: yes, makes sense

File: src/patterns/news/index.njk [hunk 7]
```diff
@@ -0,0 +1,114 @@
...
+
+You can add [related content](https://ons-design-system.netlify.app/components/related-content), which appears on the right-hand side of the landing page. Related content could include a list of categories and/or contact details for the media team. Another optional feature is “[Follow us](https://ons-design-system.netlify.app/styles/typography/#list-with-social-icons)” social media links to Instagram, Facebook, Twitter and LinkedIn.
+
+Add an image to your landing page. If you do not include an image, the system will use a placeholder. Placeholders do not look professional or support the written content to tell a story. The [articles component](/components/articles) has advice on image dimensions.
+
+{{
+    patternlibExample({ "path": "patterns/news/examples/landing/index.njk" })
+}}
+
+### Article
```
[the remark this answers is not in the document]
Comment (LauraC119) [reply src/patterns/news/index.njk:56]: I agree it can get confusing between the article pattern and component. But do we use "snippet" anywhere else. What makes a component a snippet? Might need to consider this alongside other components

[the remark this answers is not in the document]
Comment (jrbarnes9) [reply src/patterns/news/index.njk:56]: "Snippet" is just a term used in SEO which refers to the title/link and description of a page shown in a "results" view like this. 
https://yoast.com/what-is-a-snippet/

[the remark this answers is not in the document]
Comment (trevorsaint) [reply src/patterns/news/index.njk:56]: @LauraC119 is looking into this.

File: src/patterns/news/examples/tag/index.njk (hunk 2 above)
Comment (trevorsaint) [reply src/patterns/news/examples/tag/index.njk:39]: Done

**Comment by LauraC119** [comment]
As discussed, @trevorsaint my changes:

1. Change the first line to:
How to display, tag and categorise news articles. 

2. After "Use the article component to build consistent articles", change the subheading from "**Landing page**" to "**News index**"

3. Replace everything in that section with this:

This index sorts the already-published news so that the most recently posted article is at the top of the list. 

Each article listed on the news index needs to have a:

- short, clearly written title that links to the news article
- short summary, usually taken from the first paragraph of the article, which should be no more than four lines (around 150 characters)
- publication date that follows house style, for example, 23 September 2020
- thumbnail image

A list of categories appears on the right-hand side of the news index, and you could also add contact details for the media team. Another optional feature is “Follow us” social media links to Instagram, Facebook, Twitter and LinkedIn.

If you do not include an image with each article, the news index will use a placeholder. Placeholders do not look professional or support the written content to tell a story. The articles component has advice on image dimensions.

4. The subheadings under **How to use this pattern** need to be these in this order:

**News index
News article
Categories
Tags**

5. Change first paragraph under **News article** to:
This is a text page that can include images and videos. Each news article is reached through a link on the news index. It needs to have at least one of the categories: statement or press release.

6. Under **Tags**, change "landing page summary" to "news index"

7. Under **Categories**:
Change "side of the landing page" to "side of the article and the news index"
Change "visible on the landing page" to "visible on the news index page"
Change "visible on the landing page preview" to "visible for each article in the news index"

**Comment by LauraC119** [comment]
And one more change:
Change "It will stand out on the landing page and it links users to the article. It needs to:" to "It will stand out on the news index and it links users to the article. It needs to:"

**Review by LauraC119 — APPROVED** [review]

File: src/patterns/news/index.njk [hunk 8]
```diff
@@ -0,0 +1,87 @@
...
+
+## When to use this pattern
+
+Use this pattern to publish a news article to a public-facing website.
+
+## How to use this pattern
+
+Use the [articles component](/components/articles) to build consistent articles.
+
+### News index
```
Comment (jrbarnes9) [inline src/patterns/news/index.njk:23]: Now that we're keeping this heading, I would move the main example down under the 'news index' content.

File: src/patterns/news/index.njk [hunk 9]
```diff
@@ -0,0 +1,87 @@
...
+This index sorts the already-published news so that the most recently posted article is at the top of the list.
+
+Each article listed on the news index needs to have a:
+
+- short, clearly written title that links to the [news article](/patterns/news/#news-article)
+- short summary, usually taken from the first paragraph of the article, which should be no more than four lines (around 150 characters)
+- [publication date that follows house style](https://style.ons.gov.uk/house-style/dates), for example, 23 September 2020
+- thumbnail image
+
+A list of categories appears on the right-hand side of the news index, and you could also add contact details for the media team. Another optional feature is “[Follow us](https://ons-design-system.netlify.app/styles/typography/#list-with-social-icons)” social media links to Instagram, Facebook, Twitter and LinkedIn.
```
Comment (jrbarnes9) [inline src/patterns/news/index.njk:34]: Don't think we should say "right-hand side" because that's not the case on all screen widths.  

Worth linking the word 'categories' to the section below.

File: src/patterns/news/examples/tag/index.njk [hunk 10]
```diff
@@ -0,0 +1,202 @@
...
+                        "title": 'Census 2021 myth buster – 11 things you think you know about the census but you’re wrong',
+                        "date": {
+                            "short": '7 April 2021',
+                            "iso": '2021-04-07'
+                        },
+                        "category": {
+                            "title": 'Press releases',
+                            "url": '#0'
+                        },
+                        "excerpt": 'The Office for National Statistics dispels some common myths around <em class="highlight">Census 2021</em>Census 2021</em>.'
```
Comment (jrbarnes9) [inline src/patterns/news/examples/tag/index.njk:75]:
```suggestion
                        "excerpt": 'The Office for National Statistics dispels some common myths around <em class="highlight">Census 2021</em>.'
```

**Review by jrbarnes9 — APPROVED** [review]

**Review by LauraC119 — COMMENTED** [review]
Sorry @trevorsaint just one tiny thing I missed before. 
Third para under "News index". 
"A list of categories appear on the news index" should be "A list of categories appears on the news index". "A list" is the subject and that's singular.

**Comment by trevorsaint** [comment]
> Sorry @trevorsaint just one tiny thing I missed before.
> Third para under "News index".
> "A list of categories appear on the news index" should be "A list of categories appears on the news index". "A list" is the subject and that's singular.

This has been changed. Thanks @LauraC119

**Review by LauraC119 — DISMISSED** [review]

**Review by LauraC119 — APPROVED** [review]

**Review by jrbarnes9 — APPROVED** [review]
